src/components/filter/FilterCreationDialog.tsx

```diff
@@ -56,6 +54,7 @@ export interface FilterCreationDialogProps {
         id: UUID;
         equipmentType: string;
     };
+    filterType?: { id: string; label: string };
```

Comment:
I think it should be mandatory everywhere.
```suggestion
    filterType: { id: string; label: string };
```